src/server/server_family.cc

```diff
@@ -56,6 +56,8 @@ ABSL_FLAG(string, requirepass, "",
           "If empty can also be set with DFLY_PASSWORD environment variable.");
 ABSL_FLAG(string, save_schedule, "",
           "glob spec for the UTC time to save a snapshot which matches HH:MM 24h time");
+ABSL_FLAG(bool, df_snapshot_format, true,
+          "if true, save in dragonfly-specific snapshotting format");
```

Comment:
the name is a bit misleading for a bool flag if you interpret df_ as a prefix, not the type of the format